webpack/scenes/ContentViews/Details/ContentViewDetailActions.js

```diff
@@ -59,6 +60,14 @@ const getContentViewDetails = (cvId, extraParams = {}) => get({
   url: api.getApiUrl(`/content_views/${cvId}`),
 });
 
+export const getGenericContent = (pluralLabel, params) => get({
+  type: API_OPERATIONS.GET,
+  key: genericContentKey(pluralLabel),
+  url: api.getApiUrl(`/${pluralLabel}`),
+  params,
+  errorToast: error => __(`Something went wrong while fetching ${lowerCase(pluralLabel)}! ${getResponseErrorMsgs(error.response)}`),
+});
```

Comment:
I'm not sure about having "generic" as a prefix to the the variables/functions you have here and elsewhere. I say that because, in the back end, Ansible Collections isn't a "generic" type, which could make the naming pattern here misleading. 

In the other "generic content" UI stuff, it's just called "content". What do you think?

Reply:
Love that you are bringing the API into my frame of reference, much thank!